### ISSUE TITLE
Why not train the lm-nmt with a baseline for warmup?

### ISSUE THREAD

**Issue by pemywei** [teslacool/SCA#6]
As you claimed, you train your lm-nmt from scratch, but why do you not use a pretrained nmt model for warmup? Can you give some experimental results about the latter strategy?

**Comment by teslacool** [comment]
Sorry, I do not understand what you mean.

Whether  it it in our paper or on readme of github, we say that we first pretrain two language models, and then use them to train nmt models from scratch.

Do you mean to use a pretrained nmt model for warmup? We has provide this option `--load-nmt`. Because of lack of time, we did not get those results.

**Comment by pemywei** [comment]
> Sorry, I do not understand what you mean.
> 
> Whether it it in our paper or on readme of github, we say that we first pretrain two language models, and then use them to train nmt models from scratch.
> 
> Do you mean to use a pretrained nmt model for warmup? We has provide this option `--load-nmt`. Because of lack of time, we did not get those results.

**Comment by pemywei** [comment]
OK. Thanks a lot. I have another question, in my experiment, the training speed of lm-nmt is much lower than that of pure nmt model, is it common?

**Comment by teslacool** [comment]
Because every forward procedure, the models have been deeper than a pure one. So the training speed will be slower. And in my exps, the training epochs needed for modes to be convergent  are also increased, so are other data augmentation methods.
